catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -117,6 +124,17 @@ public ContentDirectoryMonitor(CamelContext camelContext, int maxRetries,
         this.maxRetries = maxRetries;
         this.delayBetweenRetries = delayBetweenRetries;
         this.configurationExecutor = configurationExecutor;
+        setBlacklist();
+    }
+
+    private void setBlacklist() {
+        String badFileProperty = System.getProperty("bad.files");
```

Comment:
`bad.files` and `bad.file.extensions` are existing properties in the `system.properties` file for  karaf.  These are automatically made available system properties during the karaf boot process.  We've decided to use these properties because other endpoints use the same list; we want the CDM to follow the same conventions when validating files for ingest.